### DISCUSSION TITLE
Login Page Changes in Release 5.1

### DISCUSSION THREAD

**Discussion by siddhishinde0723** [Sunbird-Ed/Community#694] [General Discussion]
Hello Team,

We are currently in the process of deploying Release 5.1, and I have noticed a significant change in the login page URL. It appears that the URL field has been removed entirely in this release.

As part of our application's functionality, we have made modifications for certain users to directly access the login page upon opening the application. However, despite implementing the necessary changes to accommodate this requirement, we are still experiencing issues with the login page's UI, which remains unchanged.

Could you kindly provide me with detailed information regarding the changes made to the login page in Release 5.1? Additionally, I would appreciate any guidance or support you can offer to ensure that our modified login functionality aligns correctly with the updated UI.

Thank you for your assistance in this matter.

cc: @AmiableAnil @swayangjit @NavKumarV @Ajoymaity @subranil

**Comment by swayangjit** [comment]
@siddhishinde0723 In 5.1 we have removed the keycloak webpage-based login and introduced in-app login . So you can use [signin page](https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/blob/release-5.1.0.9_RC1/src/app/sign-in/sign-in.page.ts) and use it according to your requirement and make sure portal 5.1.0 is deployed as your backend service.

**Reply by siddhishinde0723** [reply]
@swayangjit Thank you will check.

**Reply by siddhishinde0723** [reply]
@swayangjit As you mention that the keycloak is been removed from mobile application in release 5.1 is this mention in release notes or is this documented anywhere?

cc: @baljeetsingh

**Reply by swayangjit** [reply]
@siddhishinde0723 Please find this [link](https://ed.sunbird.org/v/unsupported-versions/use/releases/release-notes/release-4.10.0#hot-fix-5.0.1-27-9-2022) . It is there in 5.0.1 version

**Comment by anisha147** [comment]
Hi @siddhishinde0723 

We hope the question was resolved for you. If so, could you please close this thread from your end?

cc: @swayangjit
